manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager
+PackageVersion: 4.0.19.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://efulfillment.autodesk.com/NetSWDLD/2023/ACD/EC2FCD9E-AC4D-3E9D-8526-96832A231455/SFX/DWGTrueView_2023_English_64bit_dlm.sfx.exe?authparam=b91ba1b191a933c9c10b81c98913132b812b1b811b991b3389b98913a1c933b9&SESSION_ID=123456789;1083219296;2356937597;1;authparam;SESSION_ID

```

Comment:
The URL contains an authparam and a sessionID, which will likely expire and block future installations